models/user.go

```diff
@@ -265,7 +265,7 @@ func FindUserByConfirmationToken(tx *storage.Connection, token string) (*User, e
 
 // FindUserByEmailAndAudience finds a user with the matching email and audience.
 func FindUserByEmailAndAudience(tx *storage.Connection, instanceID uuid.UUID, email, aud string) (*User, error) {
-	return findUser(tx, "instance_id = ? and email = ? and aud = ?", instanceID, email, aud)
+	return findUser(tx, "instance_id = ? and LOWER(email) = ? and aud = ?", instanceID, strings.ToLower(email), aud)
```

Comment:
I have a feeling that doing this might cause some existing accounts to be disabled or not retrieved if it's added. 
If the current database contains `abc@gmail.com` and `ABC@gmail.com`, `findUser` will only retrieve the first email.

@awalias let me know your thoughts on this

Reply:
the `LOWER(email)` should also match `ABC@gmail.com` in the db

Reply:
I agree with @kangmingtay on two existing emails with differing case. Even with the LOWER call the db should return two results (haven't tested this yet though - nursing parents with severe COVID-19), in which case which account is valid for login? There might be a need to do some accounts cleanup but that would be assuming that each email with different cases belongs to the same user (this should be so if email was validated - two different users won't receive the confirmation email even with different letter casing)